docs/en/concepts-and-designs/kubesphere-monitoring.md

```diff
@@ -0,0 +1,43 @@
+# KubeSphere Monitoring
+
+## Overview
+
+The KubeSphere monitoring system comprises many components that work together to achieve overall monitoring functionality. KubeSphere defaults to installing the monitoring module. It will deploy  or create a few components within the namespace `kubesphere-monitoring-system`:
+
+- Prometheus Operator: automates tasks for operating Prometheus instances.
+- Prometheus Servers: scrapes metrics and provides monitoring service.
+- Kube-state-metrics: exposes kubernetes resource metrics.
```

Comment:
kube-state-metrics official website says: "kube-state-metrics is a simple service that listens to the Kubernetes API server and generates metrics about the state of the objects. " , maybe we can change it to "Generates and exposes metrics for kubernetes objects like deployment, pod, namespace etc."